misc-tools/dj_make_chroot.in

```diff
@@ -147,6 +129,29 @@ if [ "$(id -u)" != 0 ]; then
 fi
 
 [ -z "$CHROOTDIR" ] && error "No chroot directory given or default known."
+
+# If host system is Debian or Ubuntu, use its release and architecture by default
+if [ "$(lsb_release -i -s || true)" = 'Debian' ] || \
+   [ "$(lsb_release -i -s || true)" = 'Ubuntu' ]; then
+    if [ -z "$ARCH" ]; then
+        ARCH="$(dpkg --print-architecture)"
+    fi
+    # If Distro is not specified pick the host value
+    # We assume that when someone passes only the RELEASE that its valid for the host distro
```

Comment:
This comment actually doesn't match the behaviour below: RELEASE is always overwritten if DISTRO was unset. I don't have a strong opinion which way, but I think we should make sure that comments, code and usage info are in sync.